api/invite_test.go

```diff
@@ -198,6 +198,9 @@ func (ts *InviteTestSuite) TestInviteExternalGitlab() {
 			userCount++
 			w.Header().Add("Content-Type", "application/json")
 			fmt.Fprint(w, `{"name":"Gitlab Test","email":"gitlab@example.com","avatar_url":"http://example.com/avatar","confirmed_at": "2020-01-01T00:00:00.000Z"}`)
+		case "/api/v4/user/emails":
```

Comment:
This was removed here https://github.com/netlify/gotrue/commit/7875fe1e1ab3acb66b26e09acab2e4616a78b9dd#diff-34b14370bda0444543be83c769e48115L200
but now required again since we get GitLab additional emails